src/main/java/estaru/birthdaygreetings/SendBirthdayNote.java

```diff
@@ -0,0 +1,47 @@
+package estaru.birthdaygreetings;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class SendBirthdayNote implements Reminder {
+    @Override
+    public void sendBirthdayMessage(Friend friend) {
```

Comment:
Class should be responsible for one thing. Your's preparing the note and send as email. Preparing note should be excluded to another class. Then `EmailReminder` could be easy replaced with another reminder like `SmsReminder`